lisa/sut_orchestrator/azure/bicepconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "analyzers": {
+    "core": {
+      "rules": {
+        "no-unused-params": {
```

Comment:
Is there any unused params? They should be removed.

Reply:
There are no unused params.
This config file is for using experimental feature - functions, other contents are default ones. I can change the level into error.

Reply:
I changed the level into error